src/sisyphus/rabbit.clj

```diff
@@ -93,7 +108,7 @@
    (:routing-key rabbit)
    (json/generate-string message)
    {:content-type "text/plain"
-    :peristent true}))
+    :persistent true}))
```

Comment:
Ha ha. What was the impact of that typeo?

Reply:
As far as I can tell it would only matter if we wanted to cycle rabbit and have all of our messages intact. This isn't something we do often so I don't think it impacted anything.